### DISCUSSION TITLE
Add an extension to the "muted" mod

### DISCUSSION THREAD

**Discussion by Nepp0** [ppy/osu#16747] [Ideas] [answered]
so I think it's a good idea to have an extension that makes the hard parts of the map louder, so like if you reach a higher star section in a map (or maybe it would be based off of pp) then that section would have higher volume and the slower and more easy parts would be more quiet.
and you'd be able to adjust the highest and lowest possible rate before starting the map

**Comment by mk56-spn** [comment]
implentation here sounds nightmarish since muted would have to interact with the star rating system ( unless idk it can maybe be pulled from the map strain chart thats on the song progress bar ingame)

**Comment by smoogipoo** [comment, marked as answer]
Don't think this is going to happen.
